ivy.xml

```diff
@@ -7,7 +7,7 @@
  <dependencies>
   <dependency org="commons-codec" name="commons-codec" rev="1.7" />
   <dependency org="commons-httpclient" name="commons-httpclient" rev="3.1" />
-  <dependency org="dom4j" name="dom4j" rev="1.5.2" />  
+  <dependency org="org.dom4j" name="dom4j" rev="2.1.1" />  
```

Comment:
Why are we not using any variable for the version? like https://github.com/Zimbra/zm-sync-store/pull/13/files#diff-170d4d8509252f1e6690c04aa302a8d1R11

This might help us to avoid this kind of big change to just update package version in future.

Reply:
Have used variable in all the zimlets from the zm-zcs  branch that is made global 
https://github.com/Zimbra/zm-zcs/blob/feature/third-party-upgrade/ivysettings.xml
line 6 and 7 (dom4j and guava)